tests/test_login.py

```diff
@@ -0,0 +1,108 @@
+import requests
+import unittest
+from db import referee_id_connection
+from decouple import config
+
+TEST_URL = config("TEST_URL", default="https://referite-6538ffaf77b0.herokuapp.com/")
+AUDIENCE_TOKEN = config("AUDIENCE_TOKEN")
+
+class TestLogin(unittest.TestCase):
+    def setUp(self):
+        self.url = TEST_URL
+        self.audienceToken = AUDIENCE_TOKEN
+
+    def get_handler(self, url, token):
+        return requests.get(
+            f'{self.url}{url}',
+            headers={
+                'authorization': token
+            }
+        )
+
+    def post_handler(self, url, data):
```

Comment:
include more header too!

ex.
```py
def post_handler(self, url, data, extra_header={}):
            return requests.post(
             f'{self.url}{url}',
             data=data,
             headers={
                 'Content-Type': 'application/x-www-form-urlencoded',
                 **extra_header
             }
         )

Reply:
its only one post method that use additional header and adding it might lead to the misunderstanding of the function of this test

Reply:
![image](https://github.com/Referite/Referite-Backend/assets/93331101/b40c5f0a-4215-4a62-9c56-01f5919487a8)
its more readable

Reply:
I see, Additional what about put `get_handler` and `post_handler` in `test/utils.py` directory and import to use
for easy to same format code with other test. sorry for broken English. ty 👍

Reply:
Done please check.